llvm/CMakeLists.txt

```diff
@@ -150,12 +150,18 @@ if ("flang" IN_LIST LLVM_ENABLE_PROJECTS)
   endif()
 endif()
 
+if ("flang-rt" IN_LIST LLVM_ENABLE_RUNTIMES)
+  if (NOT "flang" IN_LIST LLVM_ENABLE_PROJECTS)
+    message(FATAL_ERROR "Flang is not enabled, but is required for the Flang-RT runtime")
+  endif ()
+endif ()
```

Comment:
It would be nice to be able to build `flang-rt` based on an existing, already-installed `flang`. I'm not talking about gratuitous version-divergence here (in practice it's for the same version or commit anyway), but it would help on a couple of fronts:
* this would make it easier to build out the runtime for cross-compilation purposes
* building flang is always close to timing out the 6h limit of the azure pipelines agents, so any split into pieces would be welcome

Reply:
Why do we even need `flang`? I don't see any Fortran files, but maybe I'm missing something.

Reply:
@h-vetinari Addressed here: https://github.com/llvm/llvm-project/blob/4299d1e131df65edc4a9e8489fec8322e9f426e7/flang-rt/README.md#runtime-onlyout-of-tree-build

@jhuber6 There is one Fortran file (`flang/module/iso_fortran_env_impl.f90`, see #95388) that is compiled into an object file which is added to the runtime library. That file is actually a workaround for a symbol that depends on flang switches (`NUMERIC_STORAGE_SIZE`), but in principle there is no reason why (parts) of the runtime should not be implemented in Fortran, just like libc++ or even libc/compiler-rt is written in C++.